### PR TITLE
Homework 1 (Wenjie Han)

### PR DESCRIPTION
**Pull request by WHAN3LSE**
I used windows to load my Git and initially encountered some problems. However, I am very glad the issue was resolved with the help from our excellent TAs. The tutorial on markdown was pretty useful and I learnt a lot.

### CODE REVIEW

**Comment by GabrielKahn** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |         |       x       |            |
| **Achievement, creativity** |       x    |              |            |
| **Ease of access**          |  x         |              |            |

This was a very nice bio. Simple and pleasant. You used all five stylistic elements, which means you went above and beyond what was asked (Excellent Achievement, creativity). Your coding style was perfectly functional and got the job done (Excellent Coding style/strategy). I thought that your presentation was good! I docked a point simply because I think that going forward their is room for improvement in how your documents look. For example, adding space between your (awesome) picture and your list. As far as ease of access, I had no problem getting this file. Great job, man.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
-# hw01
-Repository for homework 01
+# Introduction
+*I was born in China and visited three different cities in the past three years. I like playing tennis and listening to music in my spare time. Really look forward to learning about coding.*
+* Beijing 
+* Chicago
+* London
+![GitHub Logo](IMG_0028.JPG)
```
Comment (hannahem) [inline README.md:6]: Great photo!

**Review by hannahem — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |      X     |            |            |
| **Achievement, creativity** |           |     X         |            |
| **Ease of access**          |     X      |              |            |

Remarks:
I really enjoyed your brief bio! It was straightforward and to the point, with a classic style. I am impressed by your use of many formatting tools, including headers, lists, photos, italicization, and links. Your bio shows confidence in these skill areas. I would have liked to see more details about your adventures in the three different cities -- maybe some sub-lists with further details about each? Overall, great job!

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |       x    |              |            |

Remarks:

* Good incorporation of a photo in the README file. If the image isn't hosted somewhere on the internet, you can just put it in the repo

**Comment by bensoltoff** [comment]
Stylistically, it is better to leave line breaks between different components of the Markdown. For example, instead of

```
# Introduction
*I was born in China and visited three different cities in the past three years. I like playing tennis and listening to music in my spare time. Really look forward to learning about coding.*
* Beijing 
* Chicago
* London
![GitHub Logo](IMG_0028.JPG)
```

write

```
# Introduction

*I was born in China and visited three different cities in the past three years. I like playing tennis and listening to music in my spare time. Really look forward to learning about coding.*

* Beijing 
* Chicago
* London

![GitHub Logo](IMG_0028.JPG)
```

This ensures the rendered document formats the way you expect.

**Pull request closed without merging**